src/internal_modules/roc_packet/interleaver.cpp

```diff
@@ -96,7 +96,7 @@ void Interleaver::reinit_seq_() {
         send_seq_[i] = i;
     }
     for (size_t i = block_size_; i > 0; --i) {
-        const size_t j = core::fast_random_range(0, (unsigned int)i - 1);
+        const size_t j = core::secure_random_range(0, (unsigned int)i - 1);
```

Comment:
this should not be changed (see task description)

Reply:
So, the changes made to files not mentioned in the issue but calling fast_random derivatives were necessary.
`target_nocsprng` content is ignored in a default build (using openssl).